lib/js/components/Headers/SectionHeader/SectionHeader.vue

```diff
@@ -0,0 +1,383 @@
+<template>
+	<div
+		class="sectionHeader"
+		:class="{
+			'-expandable': expandable,
+			'-horizontal': mobileLayout === SECTION_HEADER_MOBILE_LAYOUTS.HORIZONTAL,
+			[sizeClass]: true,
+		}"
+	>
+		<div class="sectionHeader__wrapper">
+			<div class="sectionHeader__main">
+				<div class="sectionHeader__header">
+					<div class="sectionHeader__titleWrapper" @click="onTitleWrapperClicked">
+						<ds-icon
+							v-if="iconLeft"
+							class="sectionHeader__icon"
+							:icon="iconLeft"
+							:size="iconSize"
+						/>
+						<div class="sectionHeader__titleContainer">
+							<div v-if="eyebrow" class="sectionHeader__eyebrow">{{ eyebrow }}</div>
+							<div class="sectionHeader__title">{{ title }}</div>
+						</div>
+						<ds-icon
+							v-if="iconRight"
+							class="sectionHeader__icon"
+							:icon="iconRight"
+							:size="iconSize"
+						/>
+						<ds-icon
+							v-if="expandable"
+							class="sectionHeader__icon"
+							:icon="ICONS.FA_CHEVRON_DOWN"
+							:rotation="chevronRotation"
+							:size="iconSize"
+						/>
+					</div>
+					<div v-if="info" class="sectionHeader__info">
+						<ds-icon-button
+							:icon="ICONS.FA_CIRCLE_QUESTION"
+							:size="ICON_BUTTON_SIZES.X_SMALL"
+							:color="ICON_BUTTON_COLORS.NEUTRAL_WEAK"
+							:touchable="false"
+							@click.prevent.stop="onInfoClicked"
+						/>
+					</div>
+				</div>
+				<div v-if="$slots.default && showSlot" class="sectionHeader__slotHorizontal">
+					<slot />
+				</div>
+			</div>
+
+			<div
+				v-if="supportingText"
+				class="sectionHeader__supportingText"
+				:class="{ '-withoutPadding': !divider }"
+				>{{ supportingText }}
+			</div>
+
+			<div
+				v-if="$slots.default && showSlot"
+				class="sectionHeader__slotVertical"
+				:class="{ '-withoutPadding': !divider }"
+			>
+				<slot />
+			</div>
+		</div>
+		<ds-divider v-if="divider" />
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/spacings';
+
+.sectionHeader {
+	$root: &;
+
+	padding: 0;
+
+	&__wrapper {
+		display: flex;
+		flex-direction: column;
+		padding: 0;
+	}
+
+	&__main {
+		align-items: center;
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		padding: $space-xxxxs 0;
+	}
+
+	&__supportingText {
+		@include text-s-compact-regular;
+
+		color: $color-neutral-text;
+		padding: 0 0 $space-xs;
+
+		@media #{breakpoint-s()} {
```

Comment:
why not on mobile? i think this padding should be removed on mobile as well when the mobileLayout is horizontal

Reply:
Because on mobile we display slot below this text, so we need padding

Reply:
fuck.. 
![](https://media.tenor.com/RSl30rOF2zsAAAAC/breaking-bad-walter-white.gif)